### PR TITLE
[Botskills] Update documentation with new schema changes

### PR DESCRIPTION
**Pull request by LCarbonellRod**
### Purpose
*What is the context of this pull request? Why is it being done?*
Since a new manifest schema was released, some documentation became obsolete, so we update it.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
* Update the `connect`, `list`, `disconnect`, and `update` documentation.

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

File: tools/botskills/docs/connect.md [hunk 1]
```diff
@@ -45,23 +46,22 @@ For Skills that require other Authentication connection configuration please fol
...
 | --lgOutFolder [path]          | (OPTIONAL) Path for the LuisGen output (defaults to a 'service' folder inside your Virtual Assistant's folder)                                                                      |
-| --skillsFile [path]           | (OPTIONAL) Path to your Virtual Assistant Skills configuration file (defaults to the `skills.json` inside your Virtual Assistant's folder)                                           |
 | --resourceGroup [name]        | (OPTIONAL) Name of your Virtual Assistant's resource group in Azure (defaults to your Virtual Assistant's bot name)                                                                 |
-| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder)                                                                    |
+| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder. Also, the skills are stored here.)                                                                    |
```
Comment (dfavretto) [inline tools/botskills/docs/connect.md]: I'd recommend to remove the final point `.`

File: tools/botskills/docs/connect.md [hunk 2]
```diff
@@ -36,7 +36,8 @@ For Skills that require other Authentication connection configuration please fol
...
 |-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | -l, --localManifest \<path>   | Path to local Skill Manifest file                                                                                                                                                   |
-| -r, --remoteManifest \<url>   | URL to remote Skill Manifest                                                                                                                                                        |
+| -r, --remoteManifest \<url>   | URL to remote Skill Manifest                                                                                         |
+| -e, --endpointName \<name>  | (OPTIONAL) Name of the endpoint to connect to your assistant (case sensitive)(defaults to using the first endpoint) (Intended V2 manifests)                                   |
```
Comment (dfavretto) [inline tools/botskills/docs/connect.md]: Take into account the optional arguments have their values enclosed with `[]`, not `<>`. Look at the value `path` from parameter `--appSettingsFile` for example.

File: tools/botskills/docs/connect.md [hunk 3]
```diff
@@ -36,7 +36,8 @@ For Skills that require other Authentication connection configuration please fol
 | Option                        | Description                                                                                                                                                                         |
 |-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | -l, --localManifest \<path>   | Path to local Skill Manifest file                                                                                                                                                   |
```
Comment (dfavretto) [inline tools/botskills/docs/connect.md]: Do we still have local manifest files? I understood with the change of schema that the manifest won't be available anymore locally.

File: tools/botskills/docs/disconnect.md [hunk 4]
```diff
@@ -18,19 +18,19 @@ botskills disconnect [option]
 | --cs                          | Determine your Virtual Assistant project structure to be a csharp-like structure                                                                                            |
 | --ts                          | Determine your Virtual Assistant project structure to be a TypeScript-like structure                                                                                        |
 | --noRefresh                   | (OPTIONAL) Determine whether the model of your skills connected are not going to be refreshed (by default they are refreshed)                                               |
+| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder. Also, the skills are stored here.)
```
Comment (dfavretto) [inline tools/botskills/docs/disconnect.md]: I'd recommend to remove the final point `.`

File: tools/botskills/docs/list.md [hunk 5]
```diff
@@ -13,6 +13,6 @@ botskills list [options]
 
 | Option                   | Description                                                             |
 |--------------------------|-------------------------------------------------------------------------|
-| -f, --skillsFile [path]  | (OPTIONAL) Path to the Virtual Assistant Skills configuration file      |
+| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder. Also, the skills are stored here.)
```
Comment (dfavretto) [inline tools/botskills/docs/list.md]: I'd recommend to remove the final point `.`

File: tools/botskills/docs/update.md [hunk 6]
```diff
@@ -15,7 +15,8 @@ botskills update [options]
 | Option                        | Description                                                                                                                                                                 |
 |-------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | -l, --localManifest \<path>   | Path to local Skill Manifest file                                                                                                                                           |
-| -r, --remoteManifest \<url>   | URL to remote Skill Manifest                                                                                                                                                |
+| -r, --remoteManifest \<url>   | URL to remote Skill Manifest                                                                                |
+| -e, --endpointName \<name>  | (OPTIONAL) Name of the endpoint to connect to your assistant (case sensitive)(defaults to using the first endpoint) (Intended V2 manifests)                                   |
```
Comment (dfavretto) [inline tools/botskills/docs/update.md]: Take into account the optional arguments have their values enclosed with `[]`, not `<>`. Look at the value `path` from parameter `--appSettingsFile` for example.

File: tools/botskills/docs/update.md [hunk 7]
```diff
@@ -24,23 +25,22 @@ botskills update [options]
...
 | --lgOutFolder [path]          | (OPTIONAL) Path for the LuisGen output (defaults to a 'service' folder inside your Virtual Assistant's folder)                                                              |
-| --skillsFile [path]           | (OPTIONAL) Path to your Virtual Assistant Skills configuration file (defaults to the `skills.json` inside your Virtual Assistant's folder)                                  |
 | --resourceGroup [name]        | (OPTIONAL) Name of your Virtual Assistant's resource group in Azure (defaults to your Virtual Assistant's bot name)                                                         |
-| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder)                                                            |
+| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder. Also, the skills are stored here.)                         |
```
Comment (dfavretto) [inline tools/botskills/docs/update.md]: I'd recommend to remove the final point `.`

File: tools/botskills/docs/connect.md (hunk 3 above)
Comment (LCarbonellRod) [reply tools/botskills/docs/connect.md]: Yes! We still need to use it. I didn't read or heard about someone that ask us to remove this.

**Pull request closed without merging**